index.html

```diff
@@ -248,51 +186,89 @@ <h3><dfn>Sampling Frequency</dfn> and <dfn>Reporting Frequency</dfn></h3>
   </p>
 </section>
 
+<section> <h2>Pressure States</h2>
+  <p>
+    <dfn>Pressure states</dfn> represents the minimal set of useful states that allows web applications
+    to react to changes in compute pressure with minimal degration in quality or service, or user
+    experience.
+  </p>
+  <pre class="idl">
+    enum PressureState { "nominal", "fair", "serious", "critical" };
+  </pre>
+  <p>
+    The <dfn>PressureState</dfn> enum represents the [=pressure state=] with the following states:
+  </p>
+  <p>
+    <style>
+      @counter-style pressure-states-emoji {
+        system: cyclic;
+        symbols: "⚪" "🟢" "🟡" "🔴";
+        suffix: " ";
+      }
+      .pressure-states {
+        list-style-type: pressure-states-emoji;
+      }
+    </style>
+    <ul class="pressure-states">
+      <li>
+        <b>Nominal</b>: Work is minimal and the system is running on lower clock speed to preserve power.
+      </li>
+      <li>
+        <b>Fair</b>: The system is doing fine, everything is smooth and it can take on additional work without issues.
+      </li>
+      <li>
+        <b>Serious</b>: There is some serious pressure on the system, but it is sustainable and the system
+        is doing well, but it is getting close to its limits:
+        <ul>
+          <li>
+            Clock speed (depending on AC or DC power) is consistently high
+          </li>
+          <li>
+            Thermals are high but system can handle it
+          </li>
+        </ul>
+        At this point, if you add more work the system may move into critical.
+      </li>
+      <li>
```

Comment:
Is there anything beyond "Critical" that is planned?

Reply:
Other platforms have an additional state like "shutdown" but I don't see much utility in that, plus security and privacy review wants as few states as possible

Reply:
Got it. Thanks!